commcare_export/excel_query.py

```diff
@@ -260,7 +260,21 @@ def compile_source(worksheet):
     if data_source_jsonpath is None or isinstance(data_source_jsonpath, jsonpath.This) or isinstance(data_source_jsonpath, jsonpath.Root):
         return data_source, api_query, None
     else:
-        return data_source, api_query, Reference(str(data_source_jsonpath))
+        if value_or_root:
+            # if the jsonpath doesn't not yield a value yield the root document
```

Comment:
nit: "doesn't not" --> "doesn't" (for some reason github is not letting me add suggestions)